src/OrbitGl/OrbitApp.cpp

```diff
@@ -2221,6 +2221,8 @@ const orbit_client_protos::TimerInfo* OrbitApp::selected_timer() const {
 }
 
 void OrbitApp::SelectTimer(const orbit_client_protos::TimerInfo* timer_info) {
+  if (timer_info != nullptr && !IsTimerActive(*timer_info)) return;
```

Comment:
What happens if a timer is selected and later became inactive: Is the timer still selected? 

I mean, 
1- Is the function name still highlighted?
2- Can the user move using the arrow keys to nearer timers?

Reply:
1 - When you make a new selection, everything gets cleared (no timers are selected). 

When you're filtering by name, it becomes inactive while it disappears from the live tab. It then becomes active again (on capture window & live tab) when you remove the filter.  Previously, even if it wasn't part of the search, the timers would still be highlighted.  I found the previous behavior a bit misleading, but could be up for debate.  The alternative, would be to have a different behavior/color for inactive timers based on whether they are inactive because they are (1) outside the selection or (2) outside of the filter.

2 - yes, but testing this did discover a bug which I just pushed a change to.  Basically, you can move to any timers that are active as expected.  If you would be moving to an inactive timer though (i.e. left arrow key which would bring you out of the selection), it does nothing.

Reply:
> I found the previous behavior a bit misleading

Totally agree with you. Although _"they are inactive because they are (1) outside the selection or (2) outside of the filter"_ makes more sense to me, I think that having they grey is ok for me.

My question was more about this:
> When you make a new selection, everything gets cleared (no timers are selected).

I'm happy now :)

and 2- makes a lot of sense, I didn't think about that :P

Everything looks good.